pytrustnfe/nfse/imperial/__init__.py

```diff
@@ -20,18 +20,15 @@ def _send(certificado, method, **kwargs):
         base_url = 'https://nfe.etransparencia.com.br/rj.petropolis/webservice/aws_nfe.aspx'  # noqa
     else:
         base_url = 'https://nfehomologacao.etransparencia.com.br/rj.petropolis/webservice/aws_nfe.aspx'  # noqa
-
     xml_send = kwargs["xml"]
     path = os.path.join(os.path.dirname(__file__), 'templates')
-    soap = render_xml(path, 'SoapRequest.xml', False, soap_body=xml_send)
-
+    soap = render_xml(path, 'SoapRequest.xml', False, soap_body=xml_send.decode())
     client = HttpClient(base_url)
     response = client.post_soap(soap, 'NFeaction/AWS_NFE.%s' % method)
-
-    response, obj = sanitize_response(response)
+    response, obj = sanitize_response(response.encode('utf-8'))
```

Comment:
is it correct? 
I am getting the error **"AttributeError: 'bytes' object has no attribute 'encode'**

It seems to me that it is wrong response.encode('utf-8') happens twice one here and one in sanitize_response(). 1st one converts to bytes and second one fails to encode 'utf-8' on bytes
# Full Traceback
Traceback (most recent call last):
  File "/opt/odoo/parts/odoo/odoo/http.py", line 651, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/opt/odoo/parts/odoo/odoo/http.py", line 310, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/opt/odoo/parts/odoo/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/opt/odoo/parts/odoo/odoo/http.py", line 693, in dispatch
    result = self._call_function(**self.params)
  File "/opt/odoo/parts/odoo/odoo/http.py", line 342, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/opt/odoo/parts/odoo/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/opt/odoo/parts/odoo/odoo/http.py", line 335, in checked_call
    result = self.endpoint(*a, **kw)
  File "/opt/odoo/parts/odoo/odoo/http.py", line 937, in __call__
    return self.method(*args, **kw)
  File "/opt/odoo/parts/odoo/odoo/http.py", line 515, in response_wrap
    response = f(*args, **kw)
  File "/opt/odoo/parts/odoo/addons/web/controllers/main.py", line 938, in call_button
    action = self._call_kw(model, method, args, {})
  File "/opt/odoo/parts/odoo/addons/web/controllers/main.py", line 926, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/opt/odoo/parts/odoo/odoo/api.py", line 689, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/opt/odoo/parts/odoo/odoo/api.py", line 680, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/opt/odoo/addons/trustcode-odoo-brasil/br_nfse_paulistana/models/invoice_eletronic.py", line 259, in action_send_eletronic_invoice
    super(InvoiceEletronic, self).action_send_eletronic_invoice()
  File "/opt/odoo/addons/trustcode-odoo-brasil/br_nfse_imperial/models/invoice_eletronic.py", line 238, in action_send_eletronic_invoice
    None, xml=xml_to_send, ambiente=self.ambiente)
  File "/usr/local/lib/python3.5/dist-packages/pytrustnfe/nfse/imperial/__init__.py", line 43, in processa_rps
    return _send(certificado, 'PROCESSARPS', **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/pytrustnfe/nfse/imperial/__init__.py", line 28, in _send
    response, obj = sanitize_response(response.encode('utf-8'))
  File "/usr/local/lib/python3.5/dist-packages/pytrustnfe/xml/__init__.py", line 49, in sanitize_response
    tree = etree.fromstring(response.encode('UTF-8'), parser=parser)
AttributeError: 'bytes' object has no attribute 'encode'